app/build.gradle

```diff
@@ -24,14 +24,21 @@ android {
     }
 }
 
+repositories {
+    maven {
+        url "https://jitpack.io"

```

Comment:
tf? thats interesting

Reply:
I'll see what I can do to clean up my imports

Reply:
Found this: https://jitpack.io/